src/PowerShellEditorServices/Server/PsesLanguageServer.cs

```diff
@@ -82,10 +88,9 @@ public async Task StartAsync()
                         serviceCollection.AddPsesLanguageServices(_hostDetails);
                     })
                     .ConfigureLogging(builder => builder
-                        .AddSerilog(Log.Logger) // TODO: Set dispose to true?
+                        .ClearProviders()
                         .AddLanguageProtocolLogging()
                         .SetMinimumLevel(_minimumLogLevel))
-                    // TODO: Consider replacing all WithHandler with AddSingleton
```

Comment:
I guess you considered it, and said no! This was an ancient comment from Rob, and Patrick and I never quite figured out why he suggested we do that.

Reply:
The LSP sample does it this way. We may need to make some adjustments so that it does dynamic registration but that's really not a big deal